gr-uhd/lib/usrp_block_impl.h

```diff
@@ -227,6 +227,12 @@ class usrp_block_impl : virtual public usrp_block
 
     //! Stores the individual command handlers
     ::uhd::dict<pmt::pmt_t, cmd_handler_t> _msg_cmd_handlers;
+
+    //! Will check a command for a direction key, if it does not exist this will use
+    // the default value for the block via _direction() defined below
+    const pmt::pmt_t get_cmd_or_default_direction(pmt::pmt_t cmd);
+    //! Block direction overloaded by block impl to return "RX"/"TX" for source/sink
+    virtual const pmt::pmt_t _direction() = 0;
```

Comment:
Did you mean
```suggestion
    pmt::pmt_t get_cmd_or_default_direction(pmt::pmt_t cmd) const;
    //! Block direction overloaded by block impl to return "RX"/"TX" for source/sink
    virtual pmt::pmt_t _direction() const = 0;
```
?

If not, then please make this a change request (those methods shoulds be `const`)